Clarify that cubes do not need to be solved at the start of FMC (#15)
CarterKoala/wca-regulations-wrc: wca-guidelines.md

# <wca-title>WCA Guidelines

> <version>Version: January 1, 2025

Reviewer: ```suggestion
<version>Version: January 1, 2024
```

- E2c4+) [CLARIFICATION] Examples of symbols or combinations of symbols that are not permitted in solutions for 3x3x3 Fewest Moves: M, e, r, Y (uppercase), T, U1, Ui, U3, U-, R++, R2', R'2, L'w, 1R, 2U, 2f, 2-3r, 1Rw, 3Rw, 2-3Rw, rw, Rr, *, →, ⮐, /, ., ?, [r], [r u], [F], &lt;f&gt;, [f]', [f]2, [x], (x), (L), 2(Rw), (U)2, (U)', (R U)', (R U R'), [R, U], [R: U].
- E2e+) [CLARIFICATION] Competitors must not derive solutions from any part of the scramble sequence, and solutions should not share significant parts with the inverse scramble sequence.
- E2e++) [EXAMPLE] Example of solutions that should result in a disqualification of the attempt (DNF): solutions beginning with the same 4 or more moves as the inverse scramble sequence.

> - E3b+) [CLARIFICATION] It is not necessary for any 3x3 Cubes to be solved at the start of the attempt. 

Author: ```suggestion
- E3b+) [CLARIFICATION] It is not necessary for any 3x3X3 Cubes to be solved at the start of the attempt. 
```

> - E3b+) [CLARIFICATION] It is not necessary for any 3x3X3 Cubes to be solved at the start of the attempt. 

Reviewer: ```suggestion
- E3b+) [CLARIFICATION] It is not necessary for any 3x3x3 Cubes to be solved at the start of the attempt. 
```

> - E3b+) [CLARIFICATION] It is not necessary for any 3x3x3 Cubes to be solved at the start of the attempt. 

Reviewer 2: ```suggestion
- E3b+) [CLARIFICATION] Puzzles do not need to be solved at the start of the attempt. 
```
Alternative (simpler?) option.

Author: Using 3x3x3 Cubes is consistent with the parent regulation, but I don't feel particularly strongly either way.
